### ISSUE TITLE
Psort serialization failure (protobuf)

### ISSUE THREAD

**Issue by Onager** [log2timeline/plaso#224]
```
Traceback (most recent call last):
  File "./tools/psort.py", line 636, in <module>
    if not Main():
  File "./tools/psort.py", line 631, in Main
    tool.ProcessStorage()
  File "./tools/psort.py", line 583, in ProcessStorage
    self._ProcessStorage()
  File "./tools/psort.py", line 231, in _ProcessStorage
    time_slice=time_slice, use_time_slicer=self._use_time_slicer)
  File "/home/onager/plaso/plaso/frontend/psort.py", line 433, in ProcessStorage
    preferred_encoding=preferred_encoding)
  File "/home/onager/plaso/plaso/lib/storage.py", line 349, in __exit__
    self.Close()
  File "/home/onager/plaso/plaso/lib/storage.py", line 870, in Close
    self._WritePreprocessObject(self._pre_obj)
  File "/home/onager/plaso/plaso/lib/storage.py", line 846, in _WritePreprocessObject
    pre_obj_data = self._pre_obj_serializer.WriteSerialized(pre_obj)
  File "/home/onager/plaso/plaso/serializer/protobuf_serializer.py", line 728, in WriteSerialized
    proto = cls.WriteSerializedObject(pre_obj)
  File "/home/onager/plaso/plaso/serializer/protobuf_serializer.py", line 688, in WriteSerializedObject
    proto, u'collection_information', value)
  File "/home/onager/plaso/plaso/serializer/protobuf_serializer.py", line 175, in WriteSerializedDictObject
    cls.WriteSerializedObject(dict_proto_add, dict_key, dict_value)
  File "/home/onager/plaso/plaso/serializer/protobuf_serializer.py", line 159, in WriteSerializedObject
    proto_attribute.data = attribute_value
  File "/usr/lib/python2.7/dist-packages/google/protobuf/internal/python_message.py", line 435, in setter
    type_checker.CheckValue(new_value)
  File "/usr/lib/python2.7/dist-packages/google/protobuf/internal/type_checkers.py", line 91, in CheckValue
    raise TypeError(message)
TypeError: <generator object LoadPlugins at 0x4275e10> has type <type 'generator'>, but expected one of: (<type 'str'>,)
```

**Comment by joachimmetz** [comment]
It looks like one of the members of the preprocess object is a generator instead of a string. Based on `LoadPlugins` in the output I would say: https://github.com/log2timeline/plaso/blob/master/plaso/frontend/psort.py#L291

**Comment by Onager** [comment]
Fixed in 240470043
